packages/jaeger-ui/src/model/ddg/DdgEdge.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { TEdge } from '@jaegertracing/plexus/lib/types';
+import { DdgVertex } from './types';
+
+export default class DdgEdge implements TEdge<{}> {
```

Comment:
I have waffled on if this should be a class that `implements TEdge<{ from: DdgVertex, to: DdgVertex }>` so that edges would have an easier and direct link to its endpoints.
I have decided against it because the data structures are pretty bi-directionally entangled and this extra bi-directional connection did not seem to have as much value as the other connections do.

Reply:
I figured `implements` added clarity that this class was intended to work with a plexus type.
I am planning on converting the `DdgEdge` class to just an object.
At that point I don't need `TDdgEdge` at all, I can just use `TEdge` from plexus.
Thoughts?

Reply:
Are you going to remove this class? I think just using `TEdge` from plexus makes sense.

Reply:
SGTM I will use `TEdge`. I think that change will also simplify something else as well.